airflow/config_templates/config.yml

```
@@ -1314,6 +1314,15 @@
       type: string
       example: ~
       default: ""
+    - name: statsd_custom_client_path
+      description: |
+        If you want to utilise your own custom Statsd client set the relevant
+        module path below. Note: The module path must exist on your PYTHONPATH
+        for Airflow to pick it up
+      version_added: ~
+      type: string
+      example: ~
+      default: ""
```

```suggestion
      default: ~
```

As it is optional option so let's keep it as **null**